backend/workflowAPI/prisma/migrations/20220219143000_add_enumeration/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - The values [GITHUB_PR_MERGE,GITHUB_ISSUE_CLOSE] on the enum `TaskAction` will be removed. If these variants are still used in the database, this will fail.
+
+*/
+-- AlterEnum
+BEGIN;
+CREATE TYPE "TaskAction_new" AS ENUM ('GITHUB_PR_OPEN', 'GITHUB_ISSUE_OPEN', 'TIMER_DATE', 'TIMER_INTERVAL', 'GOOGLE_CALENDAR_NEW_EVENT', 'GOOGLE_NEW_DOC', 'GOOGLE_NEW_SHEET');
```

Comment:
Same as above.

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/42/commits/5bd497b96dbec7651fb3c60fdd9d87ffbf3a89de